environment.yml

```diff
@@ -0,0 +1,6 @@
+name: ami-ml
+channels:
+  - defaults
```

Comment:
Final detail :

I'd also add the conda-forge channel, as defaults depend on individual setups

```
channels:
  - conda-forge
  - defaults